scripts/runfabtests.sh

```diff
@@ -191,7 +191,7 @@ standard_tests=(
 )
 
 unit_tests=(
-	"getinfo_test"
+	"getinfo_test -s SERVER_ADDR GOOD_ADDR"
```

Comment:
Why is the GOOD_ADDR given as the destination address? The test is failing for verbs because of this, I'm looking into it.